prbt_support/launch/robot.launch

```diff
@@ -60,40 +63,22 @@
 
   <!-- Spawn controllers -->
   <node ns="prbt" name="controller_spawner" pkg="controller_manager" type="spawner" args="$(arg spawn_controllers)" />
-  <include file="$(find prbt_hardware_support)/launch/stop1_executor_node.launch" />
 
   <!-- Initialize driver -->
   <node pkg="rosservice" type="rosservice" name="robot_init" args="call --wait /prbt/driver/init"/>
 
-  <!-- Read modbus register specifications -->
-  <rosparam ns="/prbt/read_api_spec" command="load" file="$(arg read_api_spec_file)" />
-  <rosparam ns="/prbt/write_api_spec" command="load" file="$(arg write_api_spec_file)" if="$(arg has_operation_mode_support)" />
-
   <!-- Open modbus connection and required modules -->
-    <!-- Modbus connection -->  
-    <include file="$(find prbt_hardware_support)/launch/modbus_client.launch">
-      <arg name="modbus_server_ip" value="$(arg modbus_server_ip)" />
-    </include>
-    <!-- Run permitted -->
-    <include file="$(find prbt_hardware_support)/launch/modbus_adapter_run_permitted_node.launch" />
-
-    <!-- Brake test -->
-    <include if="$(arg has_braketest_support)" file="$(find prbt_hardware_support)/launch/brake_test.launch">
-      <arg name="write_api_spec_file" value="$(arg write_api_spec_file)" />
-    </include>
-
-    <!-- Operation Mode -->
-    <include if="$(arg has_operation_mode_support)" 
-             file="$(find prbt_hardware_support)/launch/operation_mode.launch" />
-    <node unless="$(arg has_operation_mode_support)" 
-          pkg="prbt_hardware_support" name="fake_speed_override_node" type="fake_speed_override_node"/>
```

Comment:
`fake_speed_override` node is probably necessary, if `enable_safety_interface==False`?